client/blocks/site-icon/index.tsx

```diff
@@ -30,7 +30,7 @@ type SiteIconProps = {
 	defaultIcon?: JSX.Element | null;
 };
 
-function SiteIcon( {
+export function SiteIcon( {
```

Comment:
Note to self: The `<SiteIcon>` component is exported so it can be used individually without the HOC that connects to the redux data store (see full file for reference).